tests/functional/testUtapiClient.js

```diff
@@ -0,0 +1,50 @@
+import assert from 'assert';
+import mapSeries from 'async/map';
```

Comment:
misleading because there is an async map series that does the operations in order whereas map does not guaranty order.